### DISCUSSION TITLE
다음씬으로 넘어갈 때 정보 넘기기 static, scriptable object 등 다 해봤지만 같은 문제가 발생합니다

### DISCUSSION THREAD

**Discussion by jeejul1010** [naverz/zepeto-studio-kor#32] [5. Q&A]
랭킹씬을 만들고 이전 씬에서의 정보를 가지고 보여주게 구현했는데요 에디터와 아이패드에서는 씬이 넘어갔을 때 랭킹정보가 잘 표시되지만 모바일 아이폰에서는 빈화면으로 정보가 안 뜹니다. static으로 했을 때 이런 문제가 발생해서 scriptable object도 사용해봤는데요 같은 문제가 발생하네요 짐작가시는 문제라도 있을까요? 감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 현상이 최신 월드에서도 지속적으로 발생하는지 확인 부탁 드립니다. 
감사합니다.
